samtranslator/translator/managed_policy_translator.py

```diff
@@ -32,5 +33,8 @@ def _load_policies_from_iam(self):
 
     def load(self):
         if self._policy_map is None:
-            self._load_policies_from_iam()
+            try:
+                self._load_policies_from_iam()
+            except botocore.exceptions.ClientError as error:
+                raise error
```

Comment:
Not sure I understand the purpose of just catching the error and then immediately raising it. If that is the case, why do we need the `try: except:` at all?